digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/model/ApplicationFormData.java

```diff
@@ -0,0 +1,190 @@
+	public void setFormVersion(String formVersion) {
+		this.formVersion = formVersion;
+	}
+
+	@JsonProperty("noticeSubjectCd")
+	public String getNoticeSubjectCd() {
+		return noticeSubjectCd;
+	}
+
+	@JsonProperty("noticeSubjectCd")
+	public void setNoticeSubjectCd(String noticeSubjectCd) {
+		this.noticeSubjectCd = noticeSubjectCd;
+	}
+
+	@JsonProperty("noticeTypeCd")
+	public String getNoticeTypeCd() {
+		return noticeTypeCd;
+	}
+
+	@JsonProperty("noticeTypeCd")
+	public void setNoticeTypeCd(String noticeTypeCd) {
+		this.noticeTypeCd = noticeTypeCd;
+	}
+
+	@JsonProperty("reviewApplnTypeCd")
+	public String getReviewApplnTypeCd() {
+		return reviewApplnTypeCd;
+	}
+
+	@JsonProperty("reviewApplnTypeCd")
+	public void setReviewApplnTypeCd(String reviewApplnTypeCd) {
+		this.reviewApplnTypeCd = reviewApplnTypeCd;
+	}
+
+	@JsonProperty("presentationTypeCd")
+	public String getPresentationTypeCd() {
+		return presentationTypeCd;
+	}
+
+	@JsonProperty("presentationTypeCd")
+	public void setPresentationTypeCd(String presentationTypeCd) {
+		this.presentationTypeCd = presentationTypeCd;
+	}
+
+	@JsonProperty("reviewRoleTypeCd")
+	public String getReviewRoleTypeCd() {
+		return reviewRoleTypeCd;
+	}
+
+	@JsonProperty("reviewRoleTypeCd")
+	public void setReviewRoleTypeCd(String reviewRoleTypeCd) {
+		this.reviewRoleTypeCd = reviewRoleTypeCd;
+	}
+
+	@JsonProperty("firstGivenNm")
+	public String getFirstGivenNm() {
+		return firstGivenNm;
+	}
+
+	@JsonProperty("firstGivenNm")
+	public void setFirstGivenNm(String firstGivenNm) {
+		this.firstGivenNm = firstGivenNm;
+	}
+
+	@JsonProperty("secondGivenNm")
+	public String getSecondGivenNm() {
+		return secondGivenNm;
+	}
+
+	@JsonProperty("secondGivenNm")
+	public void setSecondGivenNm(String secondGivenNm) {
+		this.secondGivenNm = secondGivenNm;
+	}
+
+	@JsonProperty("surnameNm")
+	public String getSurnameNm() {
+		return surnameNm;
+	}
+
+	@JsonProperty("surnameNm")
+	public void setSurnameNm(String surnameNm) {
+		this.surnameNm = surnameNm;
+	}
+
+	@JsonProperty("phoneNo")
+	public String getPhoneNo() {
+		return phoneNo;
+	}
+
+	@JsonProperty("phoneNo")
+	public void setPhoneNo(String phoneNo) {
+		this.phoneNo = phoneNo;
+	}
+
+	@JsonProperty("faxNo")
+	public String getFaxNo() {
+		return faxNo;
+	}
+
+	@JsonProperty("faxNo")
+	public void setFaxNo(String faxNo) {
+		this.faxNo = faxNo;
+	}
+
+	@JsonProperty("email")
+	public String getEmail() {
+		return email;
+	}
+
+	@JsonProperty("email")
+	public void setEmail(String email) {
+		this.email = email;
+	}
+
+	@JsonProperty("manualEntryYN")
+	public String getManualEntryYN() {
+		return manualEntryYN;
+	}
+
+	@JsonProperty("manualEntryYN")
+	public void setManualEntryYN(String manualEntryYN) {
+		this.manualEntryYN = manualEntryYN;
+	}
+
+	@JsonProperty("formData")
+	public String getFormData() {
+		return formData;
+	}
+
+	@JsonProperty("formData")
+	public void setFormData(String formData) {
+		this.formData = formData;
+	}
+
+}
```

Comment:
Needs a newline.

Reply:
added

Reply:
The change is not reflecting in the PR.
https://github.com/bcgov/jag-rsbc-digital-forms-vips-integration/commit/d98a4de8506fcc73f59fb70a0865bbd0cde3d402